.gitmodules

```
@@ -12,7 +12,10 @@
 	url = https://github.com/kthohr/gcem.git
 [submodule "external/csv-parser"]
 	path = external/csv-parser
-	url = https://github.com/vincentlaucsb/csv-parser
+	url = url = https://github.com/se-sic/csv-parser.git
```

```suggestion
	url = https://github.com/se-sic/csv-parser.git
```
I think this broke during a merge